api/krusty/accumulation_test.go

```diff
@@ -160,3 +163,56 @@ spec:
     secretName: cert-tls
 `)
 }
+
+func TestAccumulateResourcesErrors(t *testing.T) {
+	const errPrefixf = "accumulating resources: accumulating resources from '%s': "
+	for _, test := range []struct {
+		name, resource, errFile, errDir string
```

Comment:
nit: Could you add explain for what is difference between `errFile` and `errDir`?

Reply:
Hi @koba1t, I re-organized the error-processing code here: https://github.com/kubernetes-sigs/kustomize/pull/5177/files#diff-15a9eb3d3d62fd27ecea240ff711d887220b80ac54f3f709a7ed8bc25632a837R176 and added a comment explaining here: https://github.com/kubernetes-sigs/kustomize/pull/5177/files#diff-15a9eb3d3d62fd27ecea240ff711d887220b80ac54f3f709a7ed8bc25632a837R172. 

Hopefully these make it easier to understand. Thanks for the review!

Reply:
Thanks!
I think this new test code is very clear about what it is doing!